xmtp-inbox-ios.xcodeproj/project.pbxproj

```diff
@@ -643,7 +647,7 @@
 				CURRENT_PROJECT_VERSION = 3;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"xmtp-inbox-ios/Preview Content\"";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = FY4NZR34Z3;
```

Comment:
This is the XMTP team. Is it alright to commit? Makes it easier to load on my device to test.